app/errors.js

```diff
@@ -36,11 +36,21 @@ exports.noAlbum = {
 };
 
 exports.alreadyBought = {
-  statusCode: 404,
+  statusCode: 422,
   message: 'The user has already bought the book before.'
 };
 
 exports.albumsProviderFail = {
-  statusCode: 500,
+  statusCode: 422,
```

Comment:
I think this http code should be something in the range of the 5xx. For more information about http status codes you can read https://en.wikipedia.org/wiki/List_of_HTTP_status_codes